base_report_to_printer/models/printing_job.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016 SYLEAM (<http://www.syleam.fr>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+from openerp import models, fields, api
+
+_logger = logging.getLogger(__name__)
+
+
+class PrintingJob(models.Model):
+    _name = 'printing.job'
+    _description = 'Printing Job'
+    _order = 'jobid DESC'
+
+    name = fields.Char(help='Job name.')
+    active = fields.Boolean(
+        default=True, help='Unchecked if the job is purged from cups.')
+    jobid = fields.Integer(
```

Comment:
I did that to differentiate CUPS' jobid (which is a regular integer in Odoo) from a real `*_id` many2one Odoo field.